wuvt/db_utils.py

```diff
@@ -78,6 +78,7 @@ def add_sample_articles():
 Cras iaculis nibh eros, nec iaculis purus viverra non.
 """, published=True)
     article.front_page = True
+    article.pinned_article = True
```

Comment:
Any reason to do this with one of the existing sample articles? Maybe it'd be better to add a new one that is pinned?

Reply:
mostly because I wanted to test without having to manually edit it. I was having sort issues. I could delete the change but don't feel strongly either way

Reply:
raeda in fossa est